app/models/media_attachment.rb

```diff
@@ -21,6 +21,11 @@
 #  blurhash                    :string
 #  processing                  :integer
 #  file_storage_schema_version :integer
+#  thumbnail_file_name         :string
+#  thumbnail_content_type      :string
+#  thumbnail_file_size         :integer
+#  thumbnail_updated_at        :datetime
+#  thumbnail_remote_url        :string
```

Comment:
I'm a bit unsure about adding 5 new columns that would only be used by a few audio attachments…

Reply:
True, true. But if we went with my original approach, it would be a nightmare to allow users to upload their own thumbnail. And I feel like if we didn't add that functionality it would be very disappointing to end-users.

Reply:
At least they're all nullable so they don't require a table rewrite.